src/index.ts

```diff
@@ -432,6 +421,9 @@ function updateFileInCache(
   //
   if (!instance.rootFileNames.has(filePath)) {
     instance.version!++;
+    if (options.onlyCompileBundledFiles) {
+      instance.rootFileNames.add(filePath);
```

Comment:
Need to add the file manually, otherwise rootFileNames will just be declarations